addons/website/views/snippets/s_card_offset.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_card_offset" name="Card Offset">
+    <section class="s_card_offset o_cc o_cc1 pt64 pb64">
```

Comment:
Is `o_cc o_cc1` necessary ?

Reply:
necessary no.. personal opinion here. I like to see the things "selected"

![image](https://github.com/user-attachments/assets/d19b0543-0214-4dd6-8ad2-7f55f256c602)

wdyt?

Reply:
Ok 👍